test/domain/reaction/getByPost.spec.ts

```diff
@@ -0,0 +1,21 @@
+
+import { describe, expect, it } from 'vitest';
+import { COMMENT_MESSAGE, EXISTING_REQUESTER, POST_ID, createCommentReaction, createDatabase, getByPost, remove } from './_fixtures/reaction.fixture';
+
+describe('domain/reaction/getByPost', () =>
+{
+    it('should not retrieve deleted reactions', async () =>
+    {
+        await createDatabase();
+
+        const reaction = await createCommentReaction(EXISTING_REQUESTER, POST_ID, COMMENT_MESSAGE);
+
+        const reactions = await getByPost(EXISTING_REQUESTER, POST_ID);
+        expect(reactions).toContainEqual(reaction);
+
+        await remove(EXISTING_REQUESTER, reaction.id);
```

Comment:
This test currently also tests the remove function. Is it an option to alter the test database to also contain deleted reactions and test if they are not a part of the result instead?